src/snapshot.ts

```diff
@@ -85,7 +85,42 @@ export function absoluteToStylesheet(cssText: string, href: string): string {
   });
 }
 
+function getAbsoluteSrcsetString(doc: Document, attributeValue: string) {
+  if(attributeValue.trim() === "") {
+    return attributeValue
+  }
+  let resultingSrcsetString = ""
+
+  // srcset attributes is defined as such:
+  // srcset = "url size,url1 size1"
+  const srcsetValues = attributeValue.split(",")
+  srcsetValues.forEach((srcItem, i) => {
+    // removing all but middle spaces
+    const trimmedSrcItem = srcItem.trimLeft().trimRight()
+    const urlAndSize = trimmedSrcItem.split(" ")
+    // this means we have both 0:url and 1:size
+    if (urlAndSize.length == 2) {
+      const absUrl = absoluteToDoc(doc, urlAndSize[0])
+      resultingSrcsetString += absUrl
+      resultingSrcsetString +=  " "
+      resultingSrcsetString +=  urlAndSize[1]
+    } else if(urlAndSize.length == 1){
+      const absUrl = absoluteToDoc(doc, urlAndSize[0])
+      resultingSrcsetString += absUrl
+    }
+
+    if (i !== srcsetValues.length -1) {
```

Comment:
same here

Reply:
```ts
const resultingSrcsetString = srcsetValues.map(...).join(',')
```

This may make the code easier to read but the current implementation is right and totally ok.